app/views/idv/shared/_document_capture.html.erb

```diff
@@ -12,10 +12,13 @@
   ].
     compact.
     map { |url| URI(url).tap { |uri| uri.query = nil }.to_s }
-) %>
+)
+
+   session_id = flow_session[:document_capture_session_uuid]
```

Comment:
we don't reuse this, I'd just inline it?

Reply:
I agree it's not ideal but it's mostly there to avoid the 100 character line limit. I tried a few different formats to include it but I felt all of them were less readable than just pulling the single use variable.